core/util/src/test/scala/net/liftweb/util/PropsSpec.scala

```diff
@@ -17,16 +17,18 @@
 package net.liftweb
 package util
 
+import net.liftweb.common.Full
 import org.specs2.mutable.Specification
+import org.specs2.specification.AfterEach
 import Props.RunModes._
 
-/**
- * Systems under specification for Lift Mailer.
- */
-object PropsSpec extends Specification {
+object PropsSpec extends Specification with AfterEach {
   "Props Specification".title
   sequential
 
+  // TODO: Why doesn't this work??
+  def after = Props.testReset()
```

Comment:
Yeah, never figured out why `after` didn't do what I expected. Maybe it won't matter if I get what we want from using `RulesSeq[T]`. :)